lab_1/main.py

```diff
@@ -4,68 +4,135 @@
 """
 
 
-def tokenize(text: str) -> list or None:
+import json
+
+from os.path import exists
+from typing import Optional
+
+
+def tokenize(text: str) -> Optional[list]:
```

Comment:
Next time, please, don't change the provided interface. 
Even if you wanted to show smth
It can lead to some conflicts after

Reply:
Oh. My apologies!! Won't do this again without your permission! Really sorry.

Reply:
ok :)